### PR TITLE
DOC: GetPixel example separated, dynamic and statically typed languages.

### PR DESCRIPTION
**Pull request by zivy**

### CODE REVIEW

File: docs/source/fundamentalConcepts.rst [hunk 1]
```diff
@@ -112,11 +112,19 @@ units has `not ended well in the past <https://en.wikipedia.org/wiki/Mars_Climat
...
+      .. code-block:: python
+
+        image.GetPixel((0,0))
+
+   b. In statically typed languages such as C# and C++ use type specific functions:
+
+      .. code-block:: C++
 
-     image.GetPixel((0,0))
+        image.GetPixelAsUInt8( std::vector<uint32_t>( 0, 0 ) )
```
Comment (blowekamp) [inline docs/source/fundamentalConcepts.rst:127]: The new C++11 way is `image.GetPixelAsUInt8( {0, 0} )`.

Comment (zivy) [reply docs/source/fundamentalConcepts.rst:127]: Well, new and shiny is always better. Will modify.

Comment (zivy) [reply docs/source/fundamentalConcepts.rst:127]: Well, new and shiny is always better. Will modify.

**Review by dave3d — APPROVED** [review]
LGTM

**Pull request merged**